addons/mail/models/discuss/discuss_gif_favorite.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+class GifFavorite(models.Model):
+    _name = 'discuss.gif.favorite'
+    _description = 'Save favorite gif from tenor API'
+
+    tenor_gif_id = fields.Char('Gif id from tenor', required=True, index=True)
```

Comment:
Remove index (search is done by uid, which is covered by unique below)

Reply:
Done